example/example.client.ts

```diff
@@ -20,13 +22,31 @@ type GetV1UserRetrievePositiveResponse = {
   };
 };
 
-type GetV1UserRetrieveNegativeResponse = {
+interface GetV1UserRetrievePositiveResponseVariants {
+  200: GetV1UserRetrievePositiveVariant1;
+}
+
+type GetV1UserRetrievePositiveResponse =
+  SomeOf<GetV1UserRetrievePositiveResponseVariants>;
```

Comment:
maybe I can reduce the number of entities by allowing public interfaces to hold type expressions, not only type references

Reply:
addressed